test/unit/controllers/Cid_EntryController.test.js

```diff
@@ -233,10 +233,38 @@ describe('UNIT Cid_EntryController', () => {
   });
 
   describe('publishCentreUpdates', () => {
+    var populate;
+    beforeEach(() => {
+      populate = sinon.stub().returnsThis();
+      sinon.stub(Centres, 'find').returns({populate: populate, then: sinon.stub().resolves(true)});
+    });
+
+    afterEach(() => Centres.find.restore());
+
     var dummyMovement = [{id: 1}, {id: 2}, {id: 3}];
     it('should eventually resolve with the movements', () =>
       expect(controller.publishCentreUpdates(dummyMovement)).to.eventually.eql(dummyMovement)
     );
+
+    it('Should populate male_active_movements_in', () =>
+      controller.publishCentreUpdates()
+        .then(() => expect(Centres.find().populate).to.be.calledWith('male_active_movements_in'))
```

Comment:
you could also (maybe?) do:
`expect(populate.getCalls()).to.eql(['male_active_movements_in', 'male_active_movements_out', 'female_active_movements_in', 'female_active_movements_in'])` ...let's talk about it :)